Makefile

```diff
@@ -398,6 +398,15 @@ clippy_cuda_backend: install_rs_check_toolchain
 	RUSTFLAGS="$(RUSTFLAGS)" cargo "$(CARGO_RS_CHECK_TOOLCHAIN)" clippy --all-targets \
 		-p tfhe-cuda-backend -- --no-deps -D warnings
 
+.PHONY: check_rust_bindings_change # Check rust bindings are up to date for tfhe-cuda-backend
+check_rust_bindings_change: 
+	cargo build --release -p tfhe-cuda-backend && \
```

Comment:
for a diff you can use cargo build alone maybe ? to avoid a release build, though for bindings should be ok I guess